draft-regext-simple-contact.md

```diff
@@ -75,7 +75,10 @@ The JSON member "lang" is the same as that defined by RDAP in [@!RFC9083, Sectio
 The JSON member "masked" is a boolean. When true, this indicates that the data of the
 JSON object provided is to facilitate communication with the entity in a manner that
 hides or obfuscates the identity of the entity. This serves the same purpose as
-the vCard properties defined in [@!RFC8605].
+the vCard properties defined in [@!RFC8605]. This JSON member is not intended to
+redact contact information but rather provide a means of specifying contact
+information that is useable (e.g. a working email address) that does not yield
+the identity of the contact.
```

Comment:
Could unmasked data still not end up revealing the contact identity some way? Say, through non-obvious relationships. Curious how effective such masking would be?

Reply:
Yes, sloppy implementations abound. However, this is a gTLD requirement.